### ISSUE TITLE
Dev mode

### ISSUE THREAD

**Issue by AroonPro** [OpenSeizureDetector/Android_Pebble_SD#124]
Can we add in the preferences screen a easter egg for dev mode? 

Upon clicking 7 times on version in StartupActivity and or in options, the options menu will be expanded with show developers menu. 

In dev menu:

Show broadcasts of:
Boot, 
WearReceiver 
Including extra selection for received messages from messageClient
(would be tricky, because filter broadcasts and messageClient go through the same, simple drop may be possible) 
Upon state of boolean:
At StartupActivity will the slide with message window be added 
At MainActivity2 will the selected be options in the existing slider added.

**Comment by jones139** [comment]
Yes, that is a good idea - at one point I did have an Advanced Mode check
box which enabled more settings screens - we could use that value to change
the main UI too.

**Comment by jones139** [comment]
V5.0 now has a 'basic' mode with very simple UI and an 'advanced' mode with tabbed screens with more information.   We can add more tabs to that layout if required.  If some are very detailed, we can add a further 'developer' mode to show extra tabs in the UI
